validator/entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Paths
+AUTH_TOKEN_DIR="/root/.eth2validators"
+AUTH_TOKEN_FILE="${AUTH_TOKEN_DIR}/auth-token"
+
+# Generate JWT
+# --wallet-dir value   Path to a wallet directory on-disk for Prysm validator accounts (default: "/root/.eth2validators/prysm-wallet-v2")
+mkdir -p "${AUTH_TOKEN_DIR}"
+validator web generate-auth-token --wallet-dir=${AUTH_TOKEN_DIR} --accept-terms-of-use
+
+# Check if the file exists
+if [ -f "${AUTH_TOKEN_FILE}" ]; then
+    # Post JWT to dappmanager
+    curl -X POST "http://my.dappnode/data-send?key=token&data=http://prysm-prater.dappnode/initialize?token=$(sed -n 2p ${AUTH_TOKEN_FILE})" || echo "Error posting the auth-token"
```

Comment:
This should cause the script to fail and try again latter if it's the first time

Reply:
We should also give a documented alternative to users to get this token with the File Manager. Add a link to the docs or in the getting started to download this file. The File Manager supports pre-filled links